src/Microsoft.HttpRepl.Tests/FileSystem/RealFileSystemTests.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.HttpRepl.FileSystem;
+using Xunit;
+
+namespace Microsoft.HttpRepl.IntegrationTests.FileSystem
```

Comment:
Now that this is a unit test, this should be "Microsoft.HttpRepl.**Tests**.FileSystem"

Also, since the unit test assembly already has IVT access, can RealFileSystem be made internal again?

Reply:
Good catch. Changed.